### ISSUE TITLE
Classic Era 1.14

### ISSUE THREAD

**Issue by BigjoeHunter** [rgd87/ClassicAuraDurations#19]
Classicauradurations is no longer showing enemy debuffs or buffs correctly after Blizzard moved Classic Era to the 1.14 client. Please help with an update for the 1.14 client, this is one of the greatest addons of all time

**Comment by d87** [comment]
I tried it and it seems to be working fine in 1.14, Are you sure it's even loaded for you?

**Comment by BigjoeHunter** [comment]
It's loaded fine but isn't showing enemy buffs correctly. Some debuffs will show, but never buffs for enemies.

**Comment by BigjoeHunter** [comment]
have also confirmed the issue with two other players on 1.14, neither of them can see enemy buffs either when players duel or encounter each other in PvP.

**Comment by d87** [comment]
So it's only enemy buffs then. What allowed it to work was a bug with one of the game functions, I guess they finally fixed it. If that's the case then it'll be possible to only restore it to show what has been applied in your combat log range

**Comment by BigjoeHunter** [comment]
Yeah only enemy buffs are broken atm. Didn’t know about the previous bug, but if it could be updated to show what buffs appear in combat log that would also be huge, since right now those buffs aren’t showing.

Thanks so much for the quick responsiveness on this, really appreciate it!

**Comment by d87** [comment]
I tested it further on classic era ptr and everything works as it was, bugs and all.
Try `/dump ClassicAuraDurationsDB`, does it say true for enemy buffs? Maybe your settings got reset when moving to classic era

**Comment by BigjoeHunter** [comment]
I just tried the script, and nothing came up. I'll try to link a video link showing that buffs aren't showing even when in combat log range when someone finally logs in and agrees to duel.

**Comment by BigjoeHunter** [comment]
Was able to get most enemy buffs to show by changing settings to check "Full Aura Replacement". I did notice more error messages coming up in chat when certain spells are cast like the ZG spell power trinket. Here is a clip of what it looks like: https://streamable.com/ij8edg

Thanks again for all of your help troubleshooting this with me, aside from these error messages and some broken tooltips it seems to function
